src/gate/printer/garphml.cpp

```diff
@@ -0,0 +1,64 @@
+#include "graphml.h"
+
+std::string link_deskriptor(Link link)
+{
+	return std::to_string(link.source) + "_" + std::to_string(link.target);
```

Comment:
Are there two spaces from left border to "return"? Please follow the rule of having two spaces as a single tabulator.